src/supermarket/Gemfile.lock

```diff
@@ -8,7 +8,7 @@ PATH
       octokit (~> 4.16)
       rails (>= 6.1, < 7)
       ruby-filemagic
-      sidekiq (~> 4.2)
+      sidekiq (>= 4.2, < 7.0)
```

Comment:
@dheerajd-msys can we get a PR to requires >=6.2.2 since we want to make sure we always have that CVE fix?